templates/node/events/README.md

```diff
@@ -1,18 +1,29 @@
 # Node.js Cloud Events Function
 
-Welcome to your new Node.js function project! The boilerplate function code can be found in [`index.js`](./index.js). This function is meant to respond exclusively to [Cloud Events](https://cloudevents.io/), but you can remove the check for this in the function and it will respond just fine to plain vanilla incoming HTTP requests. Additionally, this example function is written asynchronously, returning a `Promise`. If your function does not perform any asynchronous execution, you can safely remove the `async` keyword from the function, and return raw values intead of a `Promise`.
+Welcome to your new Node.js function project! The boilerplate function code can
```

Comment:
Just a nit and could be solved later, do we want to use `Function` or `function`? I have seen both version across the files, we should probably settle on some convention (I am inclined to use capital `F`)

Reply:
Hahah! https://github.com/boson-project/faas/pull/206#pullrequestreview-524286346

As the CLI usability guru, Roland recommends lowercase. ;)

Reply:
OK 😄